drodrpg/DRODLib/Character.cpp

```diff
@@ -4500,12 +4493,42 @@ void CCharacter::SetCurrentGame(
 	//Check for a custom character.
 	ResolveLogicalIdentity(this->pCurrentGame ? this->pCurrentGame->pHold : NULL);
 
+	//Set the movement type
+	SetDefaultMovementType();
+
 	//If this NPC is a custom character with no script,
 	//then use the default script for this custom character type.
 	if (this->pCustomChar && this->commands.empty())
 		LoadCommands(this->pCustomChar->ExtraVars, this->commands);
 }
 
+//*****************************************************************************
+void CCharacter::SetDefaultMovementType()
+//Sets the character's eMovement to the appropriate type for it's identity
+{
+	switch (GetResolvedIdentity())
+	{
+		//These types can move through wall.
+		case M_SEEP:
+			eMovement = MovementType::WALL;
+		break;
+
+		//Flying types may also move over pits.
+		case M_WWING: case M_FEGUNDO:
+			eMovement = MovementType::AIR;
+		break;
+
+		case M_WATERSKIPPER:
+		case M_SKIPPERNEST:
```

Comment:
In RPG, M_SKIPPERNEST is called an Antlion; it's not a water-based entity.

I guess I'm not opposed to changing this to a water type like in DROD 3-5, but we should confirm this change in movement wouldn't break any published holds.

Reply:
Looks like this is one of those minor character inconsistencies that shows up sometimes - the nest can't swim, but it can move over water at the moment. Since monsters in DROD RPG generally don't move, I think it's safest to make Antlion NPCs not water dwelling, since "getting dropped in water" is more like to happen than "pile of dirt goes wandering around".